moveos/moveos-stdlib/src/natives/moveos_stdlib/raw_table/mod.rs

```diff
@@ -600,6 +600,11 @@ fn native_destroy_empty_box(
     let mut table_data = table_context.table_data.borrow_mut();
 
     let handle = get_table_handle(pop_arg!(args, StructRef))?;
+    if table_data.tables.contains_key(&handle)
```

Comment:
If we do not read any data from the table, the table's content will be empty, so we may still have bug #320 . We need to consider #128

Reply:
So the `Table.content.len` is not the real table length?

Reply:
The `Talbe.content` is the runtime content that only contains the loaded table data, not all data.